test/blackbox-tests/test-cases/coq/coqdep-on-rebuild.t/dune-project

```diff
@@ -1,3 +1,3 @@
-(lang dune 2.5)
+(lang dune 3.7)
 
-(using coq 0.2)
+(using coq 0.3)
```

Comment:
We will get rid of <= 0.2 soon anyway so I want to bump it to the 0.3 version. I would do latest but @ejgallego doesn't want to.

Reply:
I'm OK if you wanna bump to 0.7, my comment was more of a question than a request. BTW this bug was fixed in Dune 3.1, so indeed there is no true version that corresponds to it.